src/Confluent.Kafka/Impl/SafeConfigHandle.cs

```diff
@@ -66,10 +60,14 @@ internal static SafeConfigHandle Create()
         }
 
         internal IntPtr Dup()
-            => LibRdKafka.conf_dup(handle);
+        {
+            CheckClosedHandle();
```

Comment:
Don't like this name - it sort of implies we're testing for the affirmative. `ThrowIfHandleClosed` is my favorite currently.